pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "case-mapping"
+version = "0.1.0"
+description = "Classes to support representation of the CASE Cyber Ontology"
+authors = []
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/casework/CASE-Mapping-Python"
+packages = [
+    { include = "case_builder" }
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Enviroment :: Console",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Libraries"
+]
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
Is this how Python >= 3.9 is supposed to be spelled?

Are we supposed to enumerate Python versions in `pyproject.toml` as well as `ci.yml`?

Reply:
This was defined during the `poetry init` process but it appears to build fine without it. Removing.